medical/models/medical_patient.py

```diff
@@ -33,73 +32,90 @@ class MedicalPatient(models.Model):
+        Age computed depending of the birth date of the
         membership request
         """
         now = datetime.now()
-        if self.dob:
-            dob = fields.Datetime.from_string(self.dob)
+        for rec_id in self:
+            if rec_id.dob:
+                dob = fields.Datetime.from_string(rec_id.dob)
 
-            if self.deceased:
-                dod = fields.Datetime.from_string(self.dod)
-                delta = relativedelta(dod, dob)
-                deceased = _(' (deceased)')
+                if rec_id.deceased:
+                    dod = fields.Datetime.from_string(rec_id.dod)
+                    delta = relativedelta(dod, dob)
+                    deceased = _(' (deceased)')
+                else:
+                    delta = relativedelta(now, dob)
+                    deceased = ''
+                years_months_days = '%s%s %s%s %s%s%s' % (
+                    delta.years, _('y'), delta.months, _('m'),
+                    delta.days, _('d'), deceased
+                )
             else:
-                delta = relativedelta(now, dob)
-                deceased = ''
-            years_months_days = str(delta.years) + _('y ') + str(
-                delta.months) + _('m ') + str(delta.days) + _('d')\
-                + deceased
-        else:
-            years_months_days = _('No DoB !')
-        self.age = years_months_days
+                years_months_days = _('No DoB !')
+            rec_id.age = years_months_days
 
-    @api.one
-    def action_invalidate(self):
+    @api.multi
+    def action_invalidate(self, ):
         self.active = False
         self.partner_id.active = False
 
-    age = fields.Char(compute='_compute_age')
-    identification_code = fields.Char(
-        string='Internal Identification',
-        help='Patient Identifier provided by the Health Center.'
-        '(different from the Social Security Number)')
-    general_info = fields.Text(string='General Information')
-    dob = fields.Date(string='Date of Birth')
-    dod = fields.Datetime(string='Deceased Date')
-    active = fields.Boolean(default=True)
-    deceased = fields.Boolean()
-    partner_id = fields.Many2one(
-        comodel_name='res.partner', required=True, ondelete='cascade',
-        index=True)
-    gender = fields.Selection(
-        selection=[
-            ('m', 'Male'),
-            ('f', 'Female'),
-        ])
-    medical_center_id = fields.Many2one(
-        comodel_name='res.partner',
-        domain="[('is_institution', '=', True)]",
-        string='Medical Center'
-    )
-    marital_status = fields.Selection(
-        selection=[
-            ('s', 'Single'),
-            ('m', 'Married'),
-            ('w', 'Widowed'),
-            ('d', 'Divorced'),
-            ('x', 'Separated'),
-            ('z', 'law marriage'),
-        ])
-
     @api.model
     @api.returns('self', lambda value: value.id)
-    def create(self, vals):
+    def create(self, vals, ):
```

Comment:
Just habit as it tends to be convenient later on; but mainly with arrays/dicts, not methods. Will change